x-pack/plugins/task_manager/server/plugin.ts

```diff
@@ -286,6 +286,7 @@ export class TaskManagerPlugin
     const isServerless = this.initContext.env.packageInfo.buildFlavor === 'serverless';
 
     const defaultCapacity = getDefaultCapacity({
+      autoCalculateDefaultEchCapacity: this.config.auto_calculate_default_ech_capacity,
```

Comment:
should we update the `logger.info` message below with this config?

Reply:
Yeah that would be useful, I added it to the log message in this commit: https://github.com/elastic/kibana/commit/1deaff29d162d546dde5a7835c75e4b175a7a38f